src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Spans will now be exported even if the request was instrumented before. This
```

Comment:
this is an instrumentation library. it can produce spans, enrich them, but exporting is not owned by instrumentation.

Reply:
It also decides if the activity will be exported or not by setting `activity.IsAllDataRequested`. This is what we were doing before `activity.IsAllDataRequested = false;`. That is the reason to use export in here, let me know if this can be clarified further.